index.html

```diff
@@ -1,33 +1,46 @@
 ---
 layout: default
-desc: Netplan.io
+desc: netplan.io
```

Comment:
Is it uppercase or lowercase? If lowercase we should be constraint even at the beginning of a sentence.

Reply:
I think the url is lower and the title is upper? you?